server/src/tests/integration.rs

```diff
@@ -76,8 +77,8 @@ async fn test_on_ethereum() -> Result<()> {
     // 10. Test out the OP bridge
     use_optimism_bridge().await?;
 
-    // 11. Test out an ERC20 token
-    deploy_erc20_token().await?;
+    // 11. Test out a simple Move contract
+    deploy_move_counter().await?;
```

Comment:
So enabling Move contract deployments like this is the biggest thing that was enabled by shift to op-move AFAICT. Does it also mean that with it the ERC20 functionality is gone? Or is it just a replacement of a less complex test case with a more complex one?

Reply:
ERC20 would be replaced with Move fungible tokens. However, in the future I think we will support ERC20 exactly as is to be compatible with ERC20 and ERC721 bridges between L1 and L2.